nms/app/packages/magmalte/app/views/traffic/PolicyFlows.js

```diff
@@ -335,41 +357,32 @@ export default function PolicyFlowsEdit(props: Props) {
     props.onChange({...props.policyRule, flow_list: flowList});
   };
 
+  const flowList = props.policyRule.flow_list || [];
   return (
-    <>
-      <DialogContent
-        data-testid="networkInfoEdit"
-        className={props.dialogClass}>
-        <List>
-          <Typography
-            variant="caption"
-            display="block"
-            className={props.descriptionClass}
-            gutterBottom>
-            {"A policy's flows determines how it routes traffic"}
-          </Typography>
```

Comment:
While the use of `Typography` was non-standard to NMS, the styling that was used was closer to the design mocks than what we have currently. We'll probably need some sort of PR to transition all the text styling at once.

Reply:
Good catch here. Actually I incorrectly used a wrong Text js file. Eric from BraveUx added this -> https://github.com/magma/magma/blob/master/nms/app/packages/magmalte/app/theme/design-system/Text.js

We should be using this consistently instead of fbcnms Text or Typography.

Reply:
Now being tracked with issue #3289 

https://github.com/magma/magma/issues/3289